swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -174,76 +165,69 @@ Distributions available remotely:
   [1.2.2] jballerina version 1.2.2
   [1.2.3] jballerina version 1.2.3
   [1.2.4] jballerina version 1.2.4
+  [1.2.5] jballerina version 1.2.5
+  [1.2.6] jballerina version 1.2.6
+  [1.2.7] jballerina version 1.2.7
+  [1.2.8] jballerina version 1.2.8
 
 Swan Lake channel
 
-  [slp1b1] Preview 1
+  [slp1] Preview 1
+  [slp2] Preview 2
+  [slp3] Preview 3
+  [slp4] Preview 4
 
 Use 'ballerina help dist' for more information on specific commands.
 
 ```
 
-The star (*) indicates the active distribution.
+> **Note:** The star (*) indicates the active distribution.
 
-### Remove Distributions
+### Removing a Distribution
 
 The `ballerina dist remove <distribution>` command allows you to delete a particular distribution from your local environment. If you’ve been updating Ballerina regularly, you may have accumulated many unused distribution versions. This command helps you to clean them up.
 
 ```sh
-→ ballerina dist remove 1.0.5
-Distribution '1.0.5' successfully removed
+→ ballerina dist remove slp1
+Distribution 'slp1' successfully removed
 ```
 
-### Change the Active Distribution
+### Updating to the Latest Preview/Patch Version
 
-The `ballerina dist use <distribution>` command sets a particular distribution version as the active one.  See the following workflow.
-
-```sh
-→ ballerina dist use 1.0.4
-'1.1.0' successfully set as the active distribution
-
-→ ballerina dist list
-Distributions available locally:
+The `ballerina dist update` command updates your active distribution to the latest patch version.
 
-  [1.0.0] jballerina version 1.0.5
-* [1.0.4] jballerina version 1.1.0
-  [1.0.5] jballerina version 1.0.5
-  [1.1.0] jballerina version 1.1.0
+E.g., If the active distribution in your environment is `1.2.7` and there exists the patch version `1.2.8` in our servers, this command will fetch and set it as the active distribution.
 
-…
+```sh
+→ sudo ballerina dist update
+Fetching the latest patch distribution for 'jballerina-1.2.8' from the remote server...
+Fetching the '1.2.8' distribution from the remote server...
+Downloading 1.2.8 100% [==========================================================] 96/96 MB
+Successfully set the latest patch distribution '1.2.8' as the active distribution
 ```
 
-### Pull a Specific Distribution
+>**Note:** If If the active distribution in your environment is `slp3`, it will bump to the next Swan Lake Preview version, which is `slp4` and will update to it.
```

Comment:
Why do we specifically say about swan lake preview versions here?

Reply:
How will the future Swan Lake versions be? Should they be like "sl2" and "sl2.1"?

Reply:
Changed to 1.2.5 as per https://github.com/ballerina-platform/ballerina-dev-website/pull/1402#discussion_r510839199.